public/app/plugins/datasource/influxdb/specs/response_parser.jest.ts

```diff
@@ -85,6 +85,32 @@ describe('influxdb response parser', () => {
     });
   });
 
+  describe('SELECT response', () => {
+    var query = 'SELECT "usage_iowait" FROM "cpu" LIMIT 10';
+    var response = {
+      results: [
+        {
+          series: [
+            {
+              name: 'cpu',
+              columns: ['time', 'usage_iowait'],
+              values: [[1488465190006040638, 10.0], [1488465190006040638, 15.5], [1488465190006040638, 20.2]],
+            },
+          ],
+        },
+      ],
+    };
+
+    var result = parser.parse(query, response);
+
+    it('should return second column', () => {
+      expect(_.size(result)).toBe(3);
+      expect(result[0].text).toBe(10.0);
```

Comment:
should text not be of string type?

Reply:
As I see `result[0]` contains `{ text: 10 }`, thus we need a `.text` call.

Reply:
text property should always be a string

Reply:
@torkelo I've fixed that in the next commit. The fix seems non-strictly related to the PR—I could send it separately in another PR if you would like.